shuup/notify/base.py

```diff
@@ -78,6 +89,8 @@ def __init__(self, name, type=Type, required=True, help_text=""):
         self.type = type
         self.required = bool(required)
         self.help_text = help_text
+        self.dotted_accessor = dotted_accessor
+        self.cannot_be_bound = True if self.dotted_accessor else cannot_be_bound
```

Comment:
I made this so, because I was uncertain if the variable binding would work otherwise properly.